src/components/core.ts

```diff
@@ -116,31 +116,27 @@ export default class Core {
    */
   public set configuration(config: EditorConfig|string) {
     /**
+     * Place config into the class property
      * Process zero-configuration or with only holderId
      * Make config object
+     *
+     * @type {EditorConfig}
      */
-    if (!_.isObject(config)) {
-      config = {
-        holder: config,
-      };
-    }
+    this.config = _.isObject(config) ? {
```

Comment:
let's separate zero-config adjustment and a storing it in the class property into the two parts of code with their own comments. That will be more maintainable.